packages/ember-auto-import/ts/webpack.ts

```diff
@@ -127,8 +130,12 @@ export default class WebpackBundler extends Plugin implements Bundler {
       entry[bundle] = [
         join(stagingDir, 'l.js'),
         join(stagingDir, `${bundle}.js`),
+        // If I understand correctly, our virtual modules will need to go in here...
```

Comment:
Correct, the virtual modules would replace the existing `join(stagingDir, ...)` lines.